core/lib/common/worker.js

```diff
@@ -343,6 +343,30 @@ module.exports = class Worker {
 		this.logger.log(`DUT has rebooted & is back online`);
 	}
 
+	/**
+	 * Wait for a service to be active/inactive
+	 *
+	 * @param {string} serviceName
+	 * @param {string} state
+	 * @param {string} target
```

Comment:
Please add description for these parameters, 

Example: https://github.com/balena-os/test-helpers/blob/master/lib/helpers/worker.ts#L248